modules/core/include/opencv2/core/types.hpp

```diff
@@ -545,7 +545,8 @@ class CV_EXPORTS RotatedRect
      */
     RotatedRect(const Point2f& point1, const Point2f& point2, const Point2f& point3);
 
-    /** returns 4 vertices of the rectangle
+    /** returns 4 vertices of the original up-right(angle=0) rectangle with the same center and size.
+    ![image](pics/rotatedrect_points_angle.png)
```

Comment:
This image is too big and `RotatedRect` already have an image and a code example in the documentation.

Reply:
But that example didn't figure out the index of each point, I suggest replacing it with your example posted in the previous issue: https://github.com/opencv/opencv/issues/23335#issuecomment-1462792337

Reply:
It would be better to modify existing image and code snippet then (https://github.com/opencv/opencv/blob/f090e0eb06215c10602cf9f909c221ec8276816e/samples/cpp/tutorial_code/snippets/core_various.cpp#L42-L54). For example mark first vertex with a small circle or print vertex indices.